test/test_log_forwarding.py

```diff
@@ -0,0 +1,159 @@
+                                       '81241b8e-c875-4a22-a30e-58003ee139ae',
+                    'client': '172.71.0.215',
+                    'client:port': '172.71.0.215:36056',
+                    'client_port': 36056,
+                    'domain_name': 'gitlab.dev.singlecell.gi.ucsc.edu',
+                    'elb': 'app/azul-gitlab-alb/c051f98624e68d7e',
+                    'elb_status_code': '204',
+                    'error_reason': '-',
+                    'matched_rule_priority': '0',
+                    'received_bytes': '963',
+                    'redirect_url': '-',
+                    'request': f'POST https://gitlab.dev.singlecell.gi.ucsc.edu:443'
+                               f'/api/v4/jobs/request?chars={unescaped} HTTP/1.1',
+                    'request_creation_time': '2022-12-31T23:55:00.386000Z',
+                    'request_http_version': 'HTTP/1.1',
+                    'request_method': 'POST',
+                    'request_processing_time': '0.000',
+                    'request_uri': f'https://gitlab.dev.singlecell.gi.ucsc.edu:443'
+                                   f'/api/v4/jobs/request?chars={unescaped}',
+                    'request_uri_host': 'gitlab.dev.singlecell.gi.ucsc.edu',
+                    'request_uri_path': '/api/v4/jobs/request',
+                    'request_uri_port': 443,
+                    'request_uri_query': f'chars={unescaped}',
+                    'request_uri_scheme': 'https',
+                    'response_processing_time': '0.000',
+                    'sent_bytes': '229',
+                    'ssl_cipher': 'ECDHE-RSA-AES128-GCM-SHA256',
+                    'ssl_protocol': 'TLSv1.2',
+                    'target': '172.71.0.215',
+                    'target:port': '172.71.0.215:80',
+                    'target:port_list': '172.71.0.215:80',
+                    'target_group_arn': 'arn:aws:elasticloadbalancing:us-east-1:'
+                                        '122796619775:targetgroup/azul-gitlab-http/136c2d6db59941f6',
+                    'target_port': 80,
+                    'target_processing_time': '0.002',
+                    'target_status_code': '204',
+                    'target_status_code_list': '204',
+                    'time': '2022-12-31T23:55:00.388951Z',
+                    'trace_id': 'Root=1-63b0cbd4-7d218b82786295005dbf8b6d',
+                    'type': 'https',
+                    'user_agent': 'gitlab-runner 15.6.1 (15-6-stable; go1.18.8; linux/amd64)'
+                }]
+
+                log_file_contents = '\n'.join(input).encode('ascii')
+                self.storage_service.put(self.log_file_key, gzip.compress(log_file_contents))
+
+                with patch('sys.stdout', new=StringIO()) as stdout:
+                    LogForwardingController(app=MagicMock()).forward_logs(event)
+                    output = stdout.getvalue()
+
+                self.maxDiff = None
```

Comment:
IIRC, this is a class attribute so modifying it in a test would leak into other tests. Precedent in Azul is to set it at the class level.